wikidatarefisland/pipes/statement_filters.py

```diff
@@ -0,0 +1,14 @@
+def referenced_statement_excluder(statement):
+    return "references" not in statement
```

Comment:
This filters items that have P143 ("Imported from Wikimedia") reference as referenced, it shouldn't. It's in the AC of the pipe.

Reply:
Ah! Gotcha; I didn't think this was supposed to be in this sprint (or possibly even in this pipe?). I think I misinterpreted `Item needs a reference` from the ticket. Do you think this needs to be solved before merging? Or is "unfiltering" these an incremental improvement

Reply:
I say let's make a ticket and get it done in this sprint (hopefully) but this should be tracked. It definitely should be in this pipe, I can't see any other place.
I see it as an incremental step because we won't suggest reference for things that shouldn't but we skip too much which is fine for the first step, e.g. we skip item datatype too